### PR TITLE
Added high card function

### PR DESCRIPTION
**Pull request by vondojr**
Adding function to find the highest card in a hand

### CODE REVIEW

File: source/CardGames.py [hunk 1]
```diff
@@ -104,6 +104,15 @@ def clearHand(self):
     self.hand = []
     self.knownCards = []
 
+  def highCard(self):
```
Comment (JHerren726) [inline source/CardGames.py:107]: Everything looks great here! Code is clean and concise.

I think everyone has agreed as well to include comments next sprint so we can easily see what we are doing in each function. Great job Joe!

File: test/test_HighCard.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+from testing_base import *
+
+
+def test_HighCardFunc():
```
Comment (JHerren726) [inline test/test_HighCard.py:4]: In my opinion, the test is great. It is concise and clear and yet it is thorough. Great job!

**Review by JHerren726 — APPROVED** [review]

File: source/CardGames.py (hunk 1 above)
Comment (vondojr) [reply source/CardGames.py:107]: Sounds good. I will add that to my code in the future.

File: test/test_HighCard.py (hunk 2 above)
Comment (vondojr) [reply test/test_HighCard.py:4]: Thanks Jon!

**Review by andrewskerratt — APPROVED** [review]
Everything looks great Joe! Reassigning the highest card to curr_high is an efficient way to do things, looks good.

**Review by TMUProfLee — APPROVED** [review]

**Pull request merged**